pkg/autoscaler/metrics_provider.go

```diff
@@ -63,7 +63,8 @@ func NewMetricProvider(metricClient MetricClient) *MetricProvider {
 }
 
 // GetMetricByName implements the interface.
-func (p *MetricProvider) GetMetricByName(name types.NamespacedName, info provider.CustomMetricInfo) (*cmetrics.MetricValue, error) {
+func (p *MetricProvider) GetMetricByName(name types.NamespacedName, info provider.CustomMetricInfo,
+	metricSelector labels.Selector) (*cmetrics.MetricValue, error) {
```

Comment:
cc @markusthoemmes 

LMK what the best way to incorporate this would be?

Reply:
I've looked through the upstream changes and it seems absolutely fine for us to just not implement it at all right now.